### PR TITLE
Spinner checkmark added 

### PR DESCRIPTION
**Pull request by daanishnasir**
-Added checkmark to show inside spinner if showCheck is true and percentage is 100.
-added test to check for class 'cui-spinner-progress__inset-circle--check'

### CODE REVIEW

File: src/lib/Spinner/index.js [hunk 1]
```diff
@@ -137,9 +147,37 @@ export default function Default() {
       </div>
 
       <div className="docs-example docs-example--spacing">
-        <h3>Percentage Prop</h3>
-        <h3><code className="small">percentage:65</code></h3>
-        <Spinner percentage={65}/>
+          <h3>Percentage Prop</h3>
```
Comment (bfbiggs) [inline src/lib/Spinner/index.js]: Style doesn't match Avatar.  Review example code (missing className row and p tags).

File: src/lib/Spinner/index.js [hunk 2]
```diff
@@ -42,20 +43,24 @@ const Spinner = props => {
             <div className="cui-spinner-progress__fill"/>
             <div className="cui-spinner-progress__fill cui-spinner-progress__fix"/>
           </div>
-          <div className="cui-spinner-progress__inset-circle">
-            {(size === 36 && showPercentage) && <div className="cui-spinner-progress__percentage">{round(percentage)}</div>}
+          <div className={`cui-spinner-progress__inset-circle` +
+          `${showCheck && percentage === 100 ? ' cui-spinner-progress__inset-circle--check' : ''}`}
```
[the remark this answers is not in the document]
Comment (daanishnasir) [reply src/lib/Spinner/index.js]: done

File: src/lib/Spinner/index.js [hunk 3]
```diff
@@ -42,20 +43,24 @@ const Spinner = props => {
             <div className="cui-spinner-progress__fill"/>
             <div className="cui-spinner-progress__fill cui-spinner-progress__fix"/>
           </div>
-          <div className="cui-spinner-progress__inset-circle">
-            {(size === 36 && showPercentage) && <div className="cui-spinner-progress__percentage">{round(percentage)}</div>}
+          <div className={`cui-spinner-progress__inset-circle` +
```
[the remark this answers is not in the document]
Comment (daanishnasir) [reply src/lib/Spinner/index.js]: done

File: src/lib/Spinner/index.js [hunk 4]
```diff
@@ -42,20 +43,24 @@ const Spinner = props => {
...
+          `${showCheck && percentage === 100 ? ' cui-spinner-progress__inset-circle--check' : ''}`}
+          >
+            {(size === 36 && showPercentage && !showCheck) && <div className="cui-spinner-progress__percentage">{round(percentage)}</div>}
```
[the remark this answers is not in the document]
Comment (daanishnasir) [reply src/lib/Spinner/index.js]: done

File: src/lib/Spinner/index.spec.js [hunk 5]
```diff
@@ -71,4 +71,10 @@ describe('tests for <Spinner />', () => {
     expect(wrapper.find('.cui-spinner-progress').hasClass('cui-spinner-progress--16')).toEqual(true);
     expect(global.console.warn).toHaveBeenCalledTimes(1);
   });
+
```
[the remark this answers is not in the document]
Comment (daanishnasir) [reply src/lib/Spinner/index.spec.js]: done

File: src/lib/Spinner/index.js (hunk 1 above)
Comment (daanishnasir) [reply src/lib/Spinner/index.js]: done

File: src/lib/Spinner/index.js [hunk 6]
```diff
@@ -137,9 +151,10 @@ export default function Default() {
       </div>
 
       <div className="docs-example docs-example--spacing">
-        <h3>Percentage Prop</h3>
-        <h3><code className="small">percentage:65</code></h3>
-        <Spinner percentage={65}/>
+          <h3>
```
Comment (bfbiggs) [inline src/lib/Spinner/index.js]: Fix the prior examples in Spinner if you would.

File: src/lib/Spinner/index.js [hunk 7]
```diff
@@ -148,3 +163,69 @@ export default function Default() {
...
+*
+* @js
+
+import {Spinner} from '@collab-ui/react';
+
+export default function Default() {
+  return (
+    <div className='row'>
+
+        <div className="docs-example docs-example--spacing">
```
[the remark this answers is not in the document]
Comment (daanishnasir) [reply src/lib/Spinner/index.js]: done

File: src/lib/Spinner/index.js [hunk 8]
```diff
@@ -148,3 +163,69 @@ export default function Default() {
...
+            <p><code className="small">size=(16)</code></p>
+          </h3>
+          <Spinner
+            size={16}
+            percentage={100}
+            showCheck
+          />
+        </div>
+
+        <div className="docs-example docs-example--spacing">
```
[the remark this answers is not in the document]
Comment (daanishnasir) [reply src/lib/Spinner/index.js]: done

File: src/lib/Spinner/index.js [hunk 9]
```diff
@@ -148,3 +163,69 @@ export default function Default() {
...
+            <p><code className="small">size=(20)</code></p>
+          </h3>
+            <Spinner
+              size={20}
+              percentage={100}
+              showCheck
+            />
+        </div>
+
+        <div className="docs-example docs-example--spacing">
```
[the remark this answers is not in the document]
Comment (daanishnasir) [reply src/lib/Spinner/index.js]: done

File: src/lib/Spinner/index.js [hunk 10]
```diff
@@ -56,6 +64,7 @@ const Spinner = props => {
         `${(color && ` cui-spinner--${color}`) || ''}` +
         `${(className && ` ${className}`) || ''}`
         }
+
```
Comment (bfbiggs) [inline src/lib/Spinner/index.js]: Remove return.

File: src/lib/Spinner/index.js [hunk 11]
```diff
@@ -111,35 +125,58 @@ import {Spinner} from '@collab-ui/react';
 
 export default function Default() {
   return (
-    <div>
+    <div className="row">
 
       <div className="docs-example docs-example--spacing">
         <h3>Size Prop</h3>
-        <h3><code className="small">size=(16)</code></h3>
```
Comment (bfbiggs) [inline src/lib/Spinner/index.js]: remove header above as well.

File: src/lib/Spinner/index.js [hunk 12]
```diff
@@ -111,35 +125,58 @@ import {Spinner} from '@collab-ui/react';
...
+          <p><code className="small">Default size=(36)</code></p>
+        </h3>
         <Spinner />
       </div>
 
       <div className="docs-example docs-example--spacing">
         <h3>Color Prop</h3>
-        <h3><code className="small">color:blue</code></h3>
+        <h3>
+          <p><code className="small">color:blue</code></p>
```
Comment (bfbiggs) [inline src/lib/Spinner/index.js]: Make prop format the same as the other ones.

File: src/lib/Spinner/index.js [hunk 13]
```diff
@@ -111,35 +125,58 @@ import {Spinner} from '@collab-ui/react';
...
-        <h3>Dark <code className="small">Spinner</code></h3>
+        <h3>
+          Dark <p><code className="small">Spinner</code></p>
+        </h3>
         <Spinner />
       </div>
 
       <div className="docs-example docs-example--spacing">
         <h3>Percentage Prop</h3>
-        <h3><code className="small">percentage:65</code></h3>
```
Comment (bfbiggs) [inline src/lib/Spinner/index.js]: Remove H3 above

File: src/lib/Spinner/index.js [hunk 14]
```diff
@@ -111,35 +125,58 @@ import {Spinner} from '@collab-ui/react';
...
+      <div className="docs-example docs-example--spacing">
+        <h3>
+          <p><code className="small">Default size=(36)</code></p>
+        </h3>
         <Spinner />
       </div>
 
       <div className="docs-example docs-example--spacing">
         <h3>Color Prop</h3>
-        <h3><code className="small">color:blue</code></h3>
```
Comment (bfbiggs) [inline src/lib/Spinner/index.js]: Remove h3 above

File: src/lib/Spinner/index.js [hunk 15]
```diff
@@ -148,3 +185,69 @@ export default function Default() {
...
+
+import {Spinner} from '@collab-ui/react';
+
+export default function Default() {
+  return (
+    <div className='row'>
+
+      <div className="docs-example docs-example--spacing">
+        <h3>
+          <p><code className="small">percentage:100</code></p>
```
Comment (bfbiggs) [inline src/lib/Spinner/index.js]: make format the same

File: src/lib/Spinner/index.js (hunk 10 above)
Comment (daanishnasir) [reply src/lib/Spinner/index.js]: done

File: src/lib/Spinner/index.js (hunk 11 above)
Comment (daanishnasir) [reply src/lib/Spinner/index.js]: removed Size Prop header

File: src/lib/Spinner/index.js (hunk 13 above)
Comment (daanishnasir) [reply src/lib/Spinner/index.js]: removed h3 Percentage Prop

File: src/lib/Spinner/index.js (hunk 14 above)
Comment (daanishnasir) [reply src/lib/Spinner/index.js]: removed

File: src/lib/Spinner/index.js (hunk 12 above)
Comment (daanishnasir) [reply src/lib/Spinner/index.js]: changed to color=(blue)

File: src/lib/Spinner/index.js (hunk 15 above)
Comment (daanishnasir) [reply src/lib/Spinner/index.js]: removed and moved to description

File: src/lib/Spinner/index.js [hunk 16]
```diff
@@ -111,35 +124,55 @@ import {Spinner} from '@collab-ui/react';
...
+        <h3>
+          <p><code className="small">color=(blue)</code></p>
+        </h3>
         <Spinner color='blue'/>
       </div>
 
       <div className="docs-example docs-example--spacing cui--dark docs-example--dark">
-        <h3>Dark <code className="small">Spinner</code></h3>
+        <h3>
+          Dark <p><code className="small">Spinner</code></p>
```
Comment (bfbiggs) [inline src/lib/Spinner/index.js]: Let's just label this Dark Spinner and remove the p tag. so it would be 
```
<h3>Dark Spinner</h3>
```

Comment (daanishnasir) [reply src/lib/Spinner/index.js]: removed p and code block

**Pull request merged**